Update generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md (#908)
github/docs: content/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md

        UseKeychain yes
        IdentityFile ~/.ssh/id_rsa
      ```

>   {% note %}
> 
>   **Note:** If you are not using the passphrase and if you get an error then try removing the UseKeychain line.
>   
>   {% endnote %}

Reviewer: To render correctly, the note box needs a blank line above the opening tag, and more indentation. See [the staging deployment](https://docs-908--patch-1.herokuapp.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent). I'd also suggest simplifying the advice about `UseKeychain`:
 
```suggestion

     {% note %}

     **Note:** If you chose not to add a passphrase to your key, you should omit the `UseKeychain` line.
  
     {% endnote %}
```

  
3. Add your SSH private key to the ssh-agent and store your passphrase in the keychain. {% data reusables.ssh.add-ssh-key-to-ssh-agent %}
   ```shell
   $ ssh-add -K ~/.ssh/id_rsa
  ```
  {% note %}

>   **Note:** The `-K` option is Apple's standard version of `ssh-add`, which stores the passphrase in your keychain for you when you add an ssh key to the ssh-agent.If you leave your passphrase blank, run the command without -K option. 

Reviewer: ```suggestion
  **Note:** The `-K` option is Apple's standard version of `ssh-add`, which stores the passphrase in your keychain for you when you add an ssh key to the ssh-agent. If you chose not to add a passphrase to your key, run the command without the `-K` option. 
```
